docs/developers-guide.md

```diff
@@ -4,6 +4,33 @@ This page gives a few hints on how to develop new recipes and modules for
 dftimewolf. Start with the [architecture](architecture.md) page if you haven't
 read it already.
 
+## Installation
+
+To be able to develop you need a local installation of dfTimewolf. To install it locally, a venv is recommended.
+
+```bash
+git clone https://github.com/log2timeline/dftimewolf/
+git checkout -b "featurename"
+python3 -m venv venv
+source /home/user/dev/dftimewolf/venv/bin/activate
+```
+
+Remove the first line in the requirements.txt file containing something something simple.
+
+Then:
+
+```bash
+pip install pipenv
+pipenv install -r requirements.txt
+pipenv install -e .
+```
+
+Now you are ready to run dfTimewolf in your local envirenment:
+
+```bash
+/home/user/dev/dftimewolf/venv/bin/dftimewolf -h
```

Comment:
This will work when doing pipenv install -e. Otherwise, a user can run 

```bash
python -m dftimewolf.cli.dftimewolf_recipes -h
```

(it's favored, since it skips the whole `-e .` part, which under the hood reinstalls everything using setup.py and setuplib, and that can cause some problems)